modules/prebidServerBidAdapter.js

```diff
@@ -111,6 +111,7 @@ function PrebidServer() {
       tid: bidRequest.tid,
       max_bids: config.maxBids,
       timeout_millis: config.timeout,
+      secure: config.secure,
```

Comment:
add here as well https://github.com/prebid/Prebid.js/blob/master/src/prebid.js#L780

Reply:
Actually I think the current PR is what we want -- if it's added to the object.assign in prebid.js as a default, then Prebid Server's URL check is always bypassed. The behavior of the code here is that when the s2sConfig contains a `secure` override, it's sent, otherwise PBS detects.